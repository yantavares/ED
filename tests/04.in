10
A .
B -.
C .-
D ...
E --
L .--
R ..
S -.-
T -
! ---
1
ATACAR REBELDES!