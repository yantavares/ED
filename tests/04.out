. - . .- . .. /.. -- -. -- .-- ... -- -.- ---
* A T R C B E D L S !